### PR TITLE
develop <- task-estimation pull request

### PR DESCRIPTION
**Pull request by Rohnnyjogers**
Content for this feature branch complete, please review.

### CODE REVIEW

**Comment by r0bert0brien** [comment]
Looks good Johnny. Covers all of the main points and explains them clearly.

File: Content/task-estimation.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Sprint retro, what went well, what can be improved, any comments. 
+
+  
+
+Notes: 
+
+Story point: A story point is a unit of measurement for a user story based on factors such as complexity, effort, uncertainty, and risk.  
+
+Epic: Epics serve as large bodies of work that can be broken down into smaller and manageable tasks. 
+
```
Comment (r0bert0brien) [inline Content/task-estimation.md]: No need for the notes - we can remove this.

Comment (Rohnnyjogers) [reply Content/task-estimation.md]: Agreed

**Review by r0bert0brien — CHANGES_REQUESTED** [review]
Remove the Notes from Task Estimation

**Comment by BenDurkan1** [comment]
All good from my side

**Pull request merged**